translation/13

```diff
@@ -0,0 +1,29 @@
+In the previous lesson we created a reducer that can handle two actions, adding a new todo, and toggling an existing todo. Right now, the code to update the todo item or to create a new one is placed right inside of the todos reducer.
+
+在上节课程中，我们创建了一个可以持有两个 action 的 reducer：增加一个新的 todo 和切换已有的 todo。现在，在 todos 的 reducer 中，可以看到更新一项 todo 或创建一个新的 todo 的这样一串代码。
+
+This function is hard to understand because it makes us two different concerns, how the todos array is updated, and how individual todos are updated. This is not a problem unique to Redux. Any time a function does too many things, you want to extract other functions from it, and call them so that every function only addresses a single concern.
+
+这个方法非常难以理解，因为这里有两点让我们很困惑：todo 列表是怎么更新的以及每个独立的 todo 又是怎么更新的。这在 Redux 中不是一个独特的问题。任何时间，如果一个方法做了太多事情，我们都想从中提取出多个方法，然后再调用它们，以至每一个方法都只专注做一件事情。
```

Comment:
"This function is hard to understand because it makes us two different concerns, how the todos array is updated, and how individual todos are updated"    这里想要表达的意思应该是： 有两个不同的功能职责同时出现在一个函数当中，这让这个函数难以被理解。       而不是这里翻译的有两点让我们困惑。